Jenkinsfile

```diff
@@ -35,7 +35,11 @@ pipeline {
         }
       }
     }
-
+    stage('Report Test Code Coverage'){
```

Comment:
Looking at the ansibletest.sh, it looks like it runs tests. It's not clear that it does report test code coverage - it collects it, and I think copies it somewhere else, but "Report Code Coverare" in Conjur repos typically means sending the code coverage to CodeClimate, which I don't see. Should this stage name be changed to something like "run tests"?

Reply:
In next PR "https://github.com/cyberark/ansible-conjur-collection/pull/122" , I have extended the same script file for Code Coverage report  .